api/Request.json

```diff
@@ -213,6 +213,52 @@
             }
           }
         },
+        "init_keepalive_parameter": {
+          "__compat": {
+            "description": "<code>init.keepalive</code> parameter",
+            "spec_url": "https://fetch.spec.whatwg.org/#dom-requestinit-keepalive",
+            "tags": [
+              "web-features:fetch-keepalive"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "66"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": "15"
+              },
+              "firefox": {
+                "version_added": "preview"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "18.0.0"
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "13"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
```

Comment:
How did you determine the support status for the other browsers?

Reply:
I just copied the data for the `init.keepalive` parameter from the `fetch.json` data. They are two different ways of implementing the same thing (`Request()` constructor, versus directly inside the `fetch()` call), so they are going to be the same.

Reply:
Thank you. Did you test the `init.keepalive` parameter in Node.js, or did you assume it's supported in the same version as the parent feature?

When comparing the data here against `fetch.json`, I noticed that there is no `nodejs` entry for the subfeature there, should we add it there?

https://github.com/mdn/browser-compat-data/blob/ad4f44fa43517e7f5c0cd5267e730b03ebb4f834/api/_globals/fetch.json#L195-L233

Reply:
Sure, added. I assumed that `init.keepalive` is just supported, the same as in the `fetch()` data.